test/auth_test.rb

```diff
@@ -457,6 +457,16 @@ def app
       assert_includes response_cookie, "heroku_session=1;"
       assert_includes response_cookie,
         "heroku_session_nonce=8bb579ed-e3a4-41ed-9c1c-719e96618f71;"
+
+      assert response_cookie =~ /^heroku_user_session=(.+)$/, "it should contain heroku_user_session"
```

Comment:
[assert_match](http://apidock.com/ruby/Test/Unit/Assertions/assert_match) might help making this one shorter